internal/parser/process.go

```diff
@@ -61,16 +61,10 @@ func ProcessDirectory(directory string) []data.EffxYaml {
 	return yamls
 }
 
-func DetectServicesFromEffxYamls(files []data.EffxYaml, apiKeyString, sourceName string) error {
-	filePaths := []string{}
+func DetectServicesFromEffxYamls(filePaths []string, apiKeyString, sourceName string) error {
```

Comment:
Why did you need to change the function signature here? I think it was cleaner before

Reply:
this makes it easier for vcs connect to import and use this file. I agree it was cleaner before but vcs-connect doesn't use the data.Effxyaml datatype

Reply:
Can `vcs-connect` use `discover.DetectServices` directly?

Reply:
I haven't read through `vcs-connect` but I'd think if you want to use utilities in `parser`, you should start with `ProcessDirectory` so that you start with appropriate data types.

Reply:
did not think about that, you're right. Cool, reverted it back